watcher/src/verification_reports_collector.rs

```diff
@@ -73,12 +72,11 @@ impl NodeClient for ConsensusNodeClient {
             };
 
         // Contact node and get a VerificationReport.
-        let verifier = Verifier::default();
         let mut client = ThickClient::new(
             // TODO: Supply a chain-id to watcher?
             String::default(),
             node_url.clone(),
-            verifier,
+            [],
```

Comment:
This is pretty much the only place where we have non-test code that wants to not verify anything.